app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -127,6 +138,25 @@ export default function HostRequestSendField({
                 Reject
               </FieldButton>
             )}
+            {hostRequest.status ===
+              HostRequestStatus.HOST_REQUEST_STATUS_CONFIRMED &&
+              availableRefrences &&
+              availableRefrences.availableWriteReferencesList.find(
+                ({ hostRequestId }) =>
+                  hostRequestId === hostRequest.hostRequestId
+              ) && (
```

Comment:
```suggestion
              availableRefrences?.availableWriteReferencesList.find(
                ({ hostRequestId }) =>
                  hostRequestId === hostRequest.hostRequestId
              ) && (
```
Makes this long-arse condition marginally more readable